meca-lookup.txt

```diff
@@ -5,4 +5,5 @@
 10.1101/2023.02.15.528649=s3://transfers-elife/biorxiv_Current_Content/February_2023/17_Feb_23_Batch_1526/4a8b2c2b-6e13-1014-8842-90e5ce4b1a0a.meca
 10.1101/2022.12.09.519720=s3://transfers-elife/biorxiv_Current_Content/February_2023/18_Feb_23_Batch_1527/f2ae30f0-6c00-1014-8600-9097ca768463.meca
 10.21203/rs.3.rs-2200020/v3=s3://prod-elife-epp-meca/2200020-meca.zip
-10.21203/rs.3.rs-2197876/v1=s3://prod-elife-epp-meca/2197876-meca.zip
\ No newline at end of file
+10.21203/rs.3.rs-2197876/v1=s3://prod-elife-epp-meca/2197876-meca.zip
+10.1101/2022.11.08.515698[2]=s3://transfers-elife/biorxiv_Current_Content/March_2023/21_Mar_23_Batch_1557/e7c056c2-6c16-1014-98e5-b5f7d8af8b03.meca
```

Comment:
the square brackets with the version number in indicates to use this meca file rather than query bioRxiv api.